src/components/molecules/index.ts

```diff
@@ -32,6 +32,7 @@ export * from "./OakRoundIcon";
 export * from "./OakSecondaryButton";
 export * from "./OakSecondaryButtonAsRadio";
 export * from "./OakSecondaryLink";
```

Comment:
This uses the `OakSecondaryLink` component so the import has to be below it or it breaks all the tests